modules/core/test/v2/unit/coins/cspr.ts

```diff
@@ -152,32 +152,32 @@ describe('Casper', function() {
       };
 
       let signedTransaction = await basecoin.signTransaction(params, () => {});
       signedTransaction.should.have.property('halfSigned');
 
-      const halfSignedTxJson = JSON.parse(signedTransaction.halfSigned.txJson);
-      halfSignedTxJson.deploy.approvals.length.should.equals(1);
-      halfSignedTxJson.deploy.approvals[0].signer.toUpperCase().should.equals(sourceKeyPairObject.getAddress().toUpperCase());
-      CsprAccountLib.Utils.isValidTransactionSignature(halfSignedTxJson.deploy.approvals[0].signature, halfSignedTxJson.deploy.hash, sourceKeyPair.pub).should.equals(true);
+      const halfSignedTx = JSON.parse(signedTransaction.halfSigned.txHex);
+      halfSignedTx.deploy.approvals.length.should.equals(1);
+      halfSignedTx.deploy.approvals[0].signer.toUpperCase().should.equals(sourceKeyPairObject.getAddress().toUpperCase());
+      CsprAccountLib.Utils.isValidTransactionSignature(halfSignedTx.deploy.approvals[0].signature, halfSignedTx.deploy.hash, sourceKeyPair.pub).should.equals(true);
 
-      params.txPrebuild.txJson = signedTransaction.halfSigned.txJson;
+      params.txPrebuild.txHex = signedTransaction.halfSigned.txHex;
       params.prv = extendedBitgoKeyPair.xprv;
       signedTransaction = await basecoin.signTransaction(params, () => {});
       signedTransaction.should.not.have.property('halfSigned');
-      signedTransaction.should.have.property('txJson');
+      signedTransaction.should.have.property('txHex');
 
-      const twiceSignedTxJson = JSON.parse(signedTransaction.txJson);
-      twiceSignedTxJson.deploy.approvals.length.should.equals(2);
-      twiceSignedTxJson.deploy.approvals[0].signer.toUpperCase().should.equals(sourceKeyPairObject.getAddress().toUpperCase());
-      twiceSignedTxJson.deploy.approvals[1].signer.toUpperCase().should.equals(bitgoKeyPairObject.getAddress().toUpperCase());
+      const twiceSignedTxHex = JSON.parse(signedTransaction.txHex);
+      twiceSignedTxHex.deploy.approvals.length.should.equals(2);
+      twiceSignedTxHex.deploy.approvals[0].signer.toUpperCase().should.equals(sourceKeyPairObject.getAddress().toUpperCase());
+      twiceSignedTxHex.deploy.approvals[1].signer.toUpperCase().should.equals(bitgoKeyPairObject.getAddress().toUpperCase());
 
-      CsprAccountLib.Utils.isValidTransactionSignature(twiceSignedTxJson.deploy.approvals[0].signature, twiceSignedTxJson.deploy.hash, sourceKeyPair.pub).should.equals(true);
-      CsprAccountLib.Utils.isValidTransactionSignature(twiceSignedTxJson.deploy.approvals[1].signature, twiceSignedTxJson.deploy.hash, bitgoKeyPair.pub).should.equals(true);
+      CsprAccountLib.Utils.isValidTransactionSignature(twiceSignedTxHex.deploy.approvals[0].signature, twiceSignedTxHex.deploy.hash, sourceKeyPair.pub).should.equals(true);
+      CsprAccountLib.Utils.isValidTransactionSignature(twiceSignedTxHex.deploy.approvals[1].signature, twiceSignedTxHex.deploy.hash, bitgoKeyPair.pub).should.equals(true);
```

Comment:
Perhaps this line could be split into several, so that it can be read better (Same for similar cases)